README.md

```diff
@@ -37,6 +37,7 @@ This role expects the following variables:
  - `docker_registries_to_login`: list of maps, each map containing login credentials for a Docker
    registry. Expected keys are `username`, `password`, `email`, and `url`. The role will attempt to
    log into each registry described in the list. Default is empty.
+ - `requests_version`: if you want to explicitly install a specific version of requests library  
```

Comment:
Should you add why this (requests_version) is an option.  The explanation is currently in the defaults file but users shouldn't need to read that to know if they should use a variable. Also maybe note what version of requests we should be using atm to avoid the issue.